recipes/nlohmann_json/all/conanfile.py

```diff
@@ -13,16 +13,26 @@ class NlohmannJsonConan(ConanFile):
     license = "MIT"
     settings = "os", "compiler", "build_type", "arch"
     options = {
-        "multiple_headers": [True, False]
+        "multiple_headers": [True, False],
+        "implicit_conversions": [True, False]
     }
     default_options = {
-        "multiple_headers": False
+        "multiple_headers": False,
+        "implicit_conversions": True
     }
 
+    @property
+    def _can_disable_implicit_conversions(self):
+        return tools.Version(self.version) >= "3.9.0"
```

Comment:
so this option is not available in versions lower than this one?

Reply:
Yes, it has been added by https://github.com/nlohmann/json/pull/1559

Reply:
then this option should not be part of the package id for lower versions

Reply:
I do remove it in `config_options`, isn't it enough?

Reply:
gotcha